tasks/main.yml

```diff
@@ -54,6 +54,11 @@
     state: present
   when: timesync_mode != 1
 
+- name: Gather package facts
+  package_facts:
+    manager: auto
+  register: _timesync_package_facts
```

Comment:
You don't need to register the output from a fact module.

Reply:
I did that intentionally because otherwise the facts could change, right? We discussed it elsewhere in the past. But I see that this might be a different situation and register is not necessary.

Reply:
Fixed.